### PR TITLE
Add lms custom version since version does not match master

### PR DESCRIPTION
**Pull request by justincaldwell**
This explicitly creates a lms version (0.1.1.lms) since it's different than master 0.1.1. Merging will effect the current lms setup as it will change the version in the Gemfile.lock on lms.

### CODE REVIEW

**Comment by kidsalsa** [comment]
@przemyslaws-iRonin @szysza do you see any issues with merging this? It relates to this PR https://github.com/acima-credit/lease_management_system/pull/4639 and must be merged before it. It looks fine to me, but I wanted to get a second pair of eyes on it before I do the merge.

**Review by szysza — APPROVED** [review]
I don't see any issues with it

**Review by przemyslaws-iRonin — APPROVED** [review]

**Pull request merged**
